feat: DIA-1324: Add batch-prediction prompts endpoints

robot-ci-heartex opened the pull request:
Hi @robot-ci-heartex!

This PR was [created](https://github.com/HumanSignal/label-studio/actions/runs/10391357271) in a response to PRs in upstream repos:
- https://github.com/HumanSignal/label-studio-sdk/pull/286
- https://github.com/HumanSignal/label-studio-sdk/pull/291

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10391357271

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10391390619

fern-api[bot] committed:
Merge branch 'develop' into 'fb-DIA-1324'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10393565488

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10393565488

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10394215542

fern-api[bot] committed:
Merge branch 'develop' into 'fb-DIA-1324'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10395388646

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10395388646

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10395466724

Commit:
Manually add label-studio-sdk deps to unblock FM

pakelley commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10473570251)
> Successfully merged:  delete mode 100644 web/libs/editor/src/tags/object/Video/Video.styl

pakelley committed:
Merge branch 'develop' into 'fb-DIA-1324'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10473570251

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-DIA-1324'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10475262693

robot-ci-heartex committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10475262693

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10475341153

pakelley committed:
manually update label-studio-sdk

pakelley committed:
update poetry
